guides/common/modules/proc_destroying-virtual-machines-on-host-delete.adoc

```diff
@@ -0,0 +1,34 @@
+[id="destroying-virtual-machines-on-host-delete"]
+= Destroying virtual machines on host delete
+
+By default, when you delete the provisioned profile host from {Project}, {Project} does not destroy the actual VM on the external compute resource.
+You can configure {Project} to destroy the VM when deleting the host entry on {Project}.
+To use the CLI instead of the {ProjectWebUI}, see the xref:cli-destroying-virtual-machines-on-host-delete[].
+
+.Prerequisite
```

Comment:
Avital commented on this somewhere else and said that it always has to be plural. I am OK with singular. Your call Lena. cc @apinnick

Reply:
IBM Style Guide requires the plural, "Prerequisites", and that is our downstream docs convention. The RH modular docs reference guide says "Use a bulleted list for prerequisites and the plural heading Prerequisites, even if your procedure only has a single prerequisite." 

I could create yet another PR to fix this globally. :-)